### ISSUE TITLE
[0.9.0 staging-1644] Too many deposits (Dependent on Biome balancing)

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#17015]
DepositTerrainModule (GoldOreBlock) spawn rate (1,00E-004) is high! It will create new deposit every 10000 blocks with max deposit size 9000! It may create too tight deposits. To fix this decrease spawn rate at least to 9,26E-005 (for 20% space between deposits)
It looks like one big deposit:
![image](https://user-images.githubusercontent.com/45708377/86739483-947dd000-c03e-11ea-9878-e522d28817b9.png)
![image](https://user-images.githubusercontent.com/45708377/86738508-cf333880-c03d-11ea-9c91-b96dd4da121e.png)
![image](https://user-images.githubusercontent.com/45708377/86740692-82e8f800-c03f-11ea-97cd-58ac76fb2ac1.png)
"Seed": 393907036

**Comment by SlayksWood** [comment]
"Seed": 814189664
But in other case when I have a lot of desert and small Cold biomes:
![image](https://user-images.githubusercontent.com/45708377/86758949-df9edf80-c04c-11ea-863d-1e35377a4ae0.png)

**Comment by keegan-o** [comment]
Ore is being drastically affected by biomes, as long as biomes are being changes, we will wait to tweak ore because tweaking ore amounts is relatively easy but totally dependent on biomes.  Only thing that changed with ore itself lately is to drastically reduce gold ore quantity, copper and iron stayed the same. Once biomes are more settled we can take another little pass on ore. That said, too much is always much much better than too little, too little would actually be a critical bug.

Overall, ore balance is less about how much in the world there is and more about how localized and easy is it to extract.

**Comment by keegan-o** [comment]
Anyways, so that this issue can be closed I reduced the gold deposits causing errors so that the errors don't show up anymore, and I also made copper deposits much more highly variable and possible to be smaller. That should quite significantly reduce the amount of overall copper ore and also make it less certain when you find a deposit that it is going to be huge, similar to the change for gold. 

Copper is more in line with gold now but should usually spawn more, and sometimes a lot more. (always less than it was before the changes though)

In general, Copper deposits nearer the surface are more variable and have a higher chance of being small, deeper deposits are slightly less variable and have a higher chance of being larger.

![image](https://user-images.githubusercontent.com/14983165/86843481-75da0200-c05b-11ea-9a3c-dfcd6027edcf.png)
![image](https://user-images.githubusercontent.com/14983165/86844057-4e376980-c05c-11ea-99d3-4bb531d399b1.png)
![image](https://user-images.githubusercontent.com/14983165/86844855-84c1b400-c05d-11ea-9261-3e313c09d1fa.png)
